atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/CommitLockTest.java

```diff
@@ -157,46 +142,12 @@ public void canAcquireLockOnMultipleCellsOnSameRow(ConflictHandler conflictHandl
     }
 
     private void commitWriteWith(PreCommitCondition preCommitCondition, ConflictHandler conflictHandler) {
-        Transaction transaction = startTransaction(preCommitCondition, conflictHandler);
-        put(transaction, ROW, COLUMN, "100");
-        transaction.commit();
-    }
-
-    private Transaction startTransaction(PreCommitCondition preCommitCondition, ConflictHandler conflictHandler) {
-        ImmutableMap<TableReference, ConflictHandler> tablesToWriteWrite = ImmutableMap.of(
-                TEST_TABLE, conflictHandler, TransactionConstants.TRANSACTION_TABLE, ConflictHandler.IGNORE_ALL);
-        return new SerializableTransaction(
```

Comment:
Creating a `SerializableTransaction` directly causes timestamp errors in tests. I observed that in other tests, we use `TestTransactionManagerImpl` to create transactions instead of instantiating `SerializableTransaction` objects directly. Following this approach resolved the issue.

Reply:
non-blocking question: do you know why the errors came up? This is much closer to how we actually create transactions in production so it's a good change, but might be interesting to look at

Reply:
Unfortunately I am not sure why the errors came up. The exception was: `com.palantir.atlasdb.transaction.api.TransactionLockWatchFailedException: Start timestamp missing from map`

Reply:
yep! No worries, this was more of a learning thing. We can talk through this one when I'm back in UK :)